app/_hub/kong-inc/acl/how-to/_consumer-groups.md.md

```diff
@@ -0,0 +1,93 @@
+
+
+### Create consumer groups 
+
+1. Using the API, create a consumer group for `dev`:
+     ```bash
+     curl --request POST \
+         --url http://localhost:8001/consumer_groups \
+         --header 'Content-Type: application/json' \
+         --header 'accept: application/json' \
+         --data '{"name":"dev"}'
+     ```
+
+1. Then create a consumer group for `admin`:
+     
+     ```bash
+     curl --request POST \
+       --url http://localhost:8001/consumer_groups \
+       --header 'Content-Type: application/json' \
+       --header 'accept: application/json' \
+       --data '{"name":"admin",}'
+     ```
+
+1. Add a consumer to the `admin` group by using the UUID of the specific consumer:
+
+    ```bash
+    curl --request POST \
+      --url http://localhost:8001/consumer_groups/admin/consumers \
+      --header 'Content-Type: application/json' \
+      --header 'accept: application/json' \
+      --data '{"consumer":"8a4bba3c-7f82-45f0-8121-ed4d2847c4a4"}'
+    ```
+1. Then add a consumer to the `dev` group. 
+
+    ```bash
+    curl --request POST \
+      --url http://localhost:8001/consumer_groups/dev/consumers \
+      --header 'Content-Type: application/json' \
+      --header 'accept: application/json' \
+      --data '{"consumer":"8a4bba3c-7f82-45f0-8121-ed4d2847c4a4"}'
+    ```
+
+### Create routes 
+
+Using the Admin API and the [expressions router](/gateway/latest/key-concepts/routes/expressions/), create two routes: one that matches `GET`, `POST` and `PUT`, and one that only matches `DELETE`. 
+
+1. Create a route that matches when the method is _not_ `DELETE`:
+
+    ```bash
+    curl --request POST \
+      --url http://localhost:8001/services/example-service/routes \
+      --form-string name=devs-and-admins \
+      --form-string expression='http.path == "/example" && http.method != "DELETE"'
+    ```
+
+1. Create a route that matches when the method _is_ DELETE:
+    
+    ```bash
+    curl --request POST \
+      --url http://localhost:8001/services/example-service/routes \
+      --form-string name=only-admins \
+      --form-string expression='http.path == "/example" && http.method == "DELETE"'
+    ```
+
+
+### Set up the ACL plugin
+
+Scope the plugin to each of these routes with the respective `allow` configuration.
+
+1. Enable the ACL plugin on the `devs-and-admin` route, setting the `allow` field to accept both groups:
+
+    ```bash
+    curl -X POST http://localhost:8001/routes/devs-and-admin/plugins \
+        --data "name=acl"  \
+        --data "config.allow[]=dev"  \
+        --data "config.allow[]=admin"
+    ```
+
+1. Enable another ACL plugin instance on the `only-admins` route, setting the `allow` field set to only accept the `admin` group:
+
+    ```bash
+    curl -X POST http://localhost:8001/routes/admins-only/plugins \
+        --data "name=acl"  \
+        --data "config.allow=admin"
+    ```
```

Comment:
```suggestion
Enable another ACL plugin instance on the `only-admins` route, setting the `allow` field set to only accept the `admin` group:

<!--vale off-->
{% plugin_example %}
plugin: kong-inc/acl
name: acl
config:
  allow:
    - admin
targets:
  - route
formats:
  - curl
  - konnect
  - yaml
  - kubernetes
{% endplugin_example %}
<!--vale on-->
```

Reply:
same here -> https://github.com/Kong/docs.konghq.com/pull/6773#discussion_r1471127508

Reply:
adjusted the config here too